Update Blueprints index.rst

Niehaus opened the pull request:
Update the layout of Blueprints #1180, it should look like this example now:

![Screenshot_2020-10-29 Blueprints — Kytos SDN Platform 2020 2b1 documentation](https://user-images.githubusercontent.com/17555995/97626864-46377700-1a09-11eb-9d6b-7113bd906f76.png)

Niehaus committed:
Update Blueprints index.rst

Niehaus committed:
Create make_blueprints_table.py

hdiogenes commented on docs/blueprints/make_blueprints_table.py:
```diff
@@ -0,0 +1,65 @@
+    
+        with open(fp) as origin_file:
+            title = 'Metadata not define for :Title:'
+            status = 'Metadata not define for :Status:'
```
```suggestion
            title = ''
            status = ''
```

hdiogenes commented on docs/blueprints/make_blueprints_table.py:
```diff
@@ -0,0 +1,65 @@
+    table_header = ''
+    table_header += '+' + '-' * 23 + '+' + '-' * 82 + '+' + '-' * 20 + '+\n'
+    table_header += '|' + ' Blueprint'  + ' ' * 13 + '|' + ' Title' + ' ' * 76 + '|' + ' Status' + ' ' * 13 + '|\n'
+    table_header += '+' + '=' * 23 + '+' + '=' * 82 + '+' + '=' * 20 + '+\n'
```
Please use a global string for this, disable the lint 80-char check if necessary.

hdiogenes commented on docs/blueprints/make_blueprints_table.py:
```diff
@@ -0,0 +1,65 @@
+        title_space = 82 - (len(title))
+        status_space = 20 - (len(status))
+        table_cell += '|' + ' :doc:`'+ bp_rst + '<' + bp_rst + '/>` '  + ' ' + '|' + title + ' ' * title_space + '|' + status + ' ' * status_space + '|\n'
+        table_cell += '+' + '-' * 23 + '+' + '-' * 82 + '+' + '-' * 20 + '+\n'
```
I think it's better to just use a string template here.

Niehaus replied:
I'm still working on some changes, but I'll take a look in your suggestions in order to apply them

hdiogenes commented on docs/blueprints/index.rst:
```diff
@@ -4,8 +4,58 @@
   :glob:
 
   EP*
+
```
This can be generated in the build stage of the docs.

Niehaus committed:
Update make_blueprints_table.py

String formats improved, bp_tables.rst created and include it on index.rst. Also the call for the new script to the Makefile has been added

hdiogenes committed:
Delete bp_table.rst
